content/webapp/pages/stories.tsx

```diff
@@ -99,12 +101,18 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
     // would it be faster to skip all the fetchLinks?  Is that possible?
     const storiesPagePromise = fetchPage(client, prismicPageIds.stories);
 
+    const featuredSerial =
+      await client.client.getSingle<FeaturedSerialPrismicDocument>(
+        'featured-serial'
+      );
+    const featuredSerialId =
+      (isFilledLinkToDocument(featuredSerial?.data?.serial) &&
+        featuredSerial.data.serial.id) ||
+      featuredStoriesSeriesId; // Fallback to hardcoded default
```

Comment:
Not sure I agree, this is liable to have a very old featured story start appearing for inexplicable reasons (because we'll forget the hard-coded default exists). Or if we remember to update it, it'll mask errors in the new code path. I think we should ditch it.